### PR TITLE
Support multiple organizers in EA Client

### PR DESCRIPTION
**Pull request by lucatume**
Ticket: https://central.tri.be/issues/94954

Service ticket: https://central.tri.be/issues/94608

Service request: https://github.com/moderntribe/event-aggregator-site/pull/273

This PR:
* adds support for multiple organizers when provided by EA Service
* exposes the Date fields in FB Imports to allow starting the import from dates before `today`

### CODE REVIEW

File: src/Tribe/Aggregator/Event.php [hunk 1]
```diff
@@ -146,12 +146,20 @@ public static function translate_service_data( $item ) {
 
 		if ( ! empty( $item->organizer ) ) {
 			$event['Organizer'] = array();
-			foreach ( $organizer_field_map as $origin_field => $target_field ) {
-				if ( ! isset( $item->organizer->$origin_field ) ) {
-					continue;
+			$organizer_entries  = $item->organizer;
```
Comment (barryhughes) [inline src/Tribe/Aggregator/Event.php]: Can we rely on this always being an array (or should we add a check or cast before using in the foreach)? I'm sure this is covered, just asking to be safe.

File: src/Tribe/Aggregator/Record/Abstract.php [hunk 2]
```diff
@@ -1535,126 +1541,139 @@ public function insert_posts( $items = array() ) {
...
+								$found_organizers[ $organizer->ID ] = $organizer_data['Organizer'];
 
-							// Log this Organizer was created
-							$activity->add( 'organizer', 'created', $event['EventOrganizerID'] );
+								// Here we might need to update the Organizer depending we found something based on old code
+								if ( 'retain' === $update_authority_setting ) {
+									// When we get here we say that we skipped an Organizer
+									$activity->add( 'organizer', 'skipped', $organizer->ID );
+								} else {
+									if ( 'preserve_changes' === $update_authority_setting ) {
```
Comment (barryhughes) [inline src/Tribe/Aggregator/Record/Abstract.php]: Not related to your changes in any way, but wow! Holy long-and-deeply-nested method!

File: src/Tribe/Aggregator/Event.php (hunk 1 above)
Comment (lucatume) [reply src/Tribe/Aggregator/Event.php]: This is covered but a little more defensive programming will not hurt.

File: src/Tribe/Aggregator/Record/Abstract.php (hunk 2 above)
Comment (lucatume) [reply src/Tribe/Aggregator/Record/Abstract.php]: I know, it's also a really long class that will benefit from some love soon.

**Pull request merged**